naturblick/utils/URL+Wikipedia.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright © 2024 Museum für Naturkunde Berlin.
+// This code is licensed under MIT license (see LICENSE.txt for details)
+
+import Foundation
+
+extension URL {
+    private static let deBaseUrl = "https://de.wikipedia.org"
+    private static let enBaseUrl = "https://en.wikipedia.org"
+
+        private static func wikiName(sciname: String) -> String? {
+            let tokens = sciname.split(separator: " ").prefix(2)
+            if(tokens[0].lowercased() == "Gattung".lowercased()) {
```

Comment:
"Gattung".lowercased()

:-)

Reply:
This is a bit of self documentation. The "species" that are in fact genus were add with the prefix "Gattung". Because the data is not that clean, I decided to also match "gattung", even though that is most likely not necessary. If we change it here I would also want to change it [here](https://github.com/MfN-Berlin/naturblick-android/blob/main/app/src/main/java/berlin/mfn/naturblick/utils/Wikipedia.kt#L17)

Reply:
ok, then leave it as it is